mkallTests/MKAsyncTaskTests.m

```diff
@@ -23,7 +22,6 @@ - (void)testMKAsyncTaskNDT {
     @"name": @"Ndt",
     @"options": @{
       @"no_file_report": @YES,
-      @"net/ca_bundle_path": [MKResources caBundlePath],
```

Comment:
@lorenzoPrimi this is because the resources are now an internal implementation detail of this framework and are automatically added when you start an AsyncTask (i.e. a nettest)

Reply:
Should I remove this from the app?

Reply:
Indeed